cloudconfig/podcfg/image.go

```diff
@@ -64,15 +65,36 @@ func IsCharmBaseImage(imagePath string) bool {
 }
 
 // GetJujuOCIImagePath returns the jujud oci image path.
-func GetJujuOCIImagePath(controllerCfg controller.Config, ver version.Number) (string, error) {
-	// First check the deprecated "caas-operator-image-path" config.
+func GetJujuOCIImagePath(controllerCfg controller.Config, modelCfg *config.Config, ver version.Number) (string, error) {
+	// First check the model config.
+	if modelCfg != nil {
```

Comment:
We shouldn't be passing in a nil config; we typically don't check for nil

Reply:
I see, will refactor. Did a check here because there was one usage that GetControllerImagePath that uses this method to get the controller image path, perhaps I could expand the method for that particular usage, thanks ian!